redaxo/src/core/boot.php

```diff
@@ -153,3 +153,23 @@
     unset($REX);
     require rex_path::core(rex::isBackend() ? 'backend.php' : 'frontend.php');
 }
+
+if (!rex::isBackend() && 0 != rex_config::get('core', 'phpmailer_errormail')) {
```

Comment:
wenn in der core boot.php rex_config verwendet wird, muss ein `!isSetup`-check hinzu, da im setup rex_config noch nicht verwendet werden kann

Reply:
evtl hier drüber ein `if (isSetup) return`. Also da drunter dann alles, was erst nach setup greifen soll